packages/mobile/src/pincode/authentication.test.ts

```diff
@@ -340,6 +341,25 @@ describe(updatePin, () => {
   })
 })
 
+describe(removeStoredPin, () => {
+  it('should remove the item from keychain', async () => {
+    mockedKeychain.resetGenericPassword.mockResolvedValue(true)
+    await removeStoredPin()
+
+    expect(mockedKeychain.resetGenericPassword).toHaveBeenCalledTimes(1)
+    expect(mockedKeychain.resetGenericPassword).toHaveBeenCalledWith({ service: 'PIN' })
+  })
+  it('should throw an error if item could not be removed from keychain', async () => {
+    mockedKeychain.resetGenericPassword.mockRejectedValue('some error')
+
+    try {
+      await removeStoredPin()
+    } catch (error) {
+      expect(error).toEqual('some error')
+    }
```

Comment:
Problem with this is if it doesn't throw an error, we're not expecting anything and the test will happily pass 😅 

Here's one way to avoid this problem:
```suggestion
    expect(await removeStoredPin()).rejects.toThrowError('some error')
```

Reply:
oh you're right! i didn't really think about this before. i went through and updated all the instances of this try/catch type of assertion in this test file, there were a few.

one interesting thing i found was `rejects.toThrowError` didn't work for this particular assertion and the assertion on line 170, i had to use `rejects.toEqual`. i could verify that the tests fail if the `toEqual` value doesn't match but not really sure why the `toThrowError` doesn't work... 🤔 for me the `rejects.toEqual` assertion is okay enough here, but if you have any ideas on why `toThrowError` doesn't work as expected then i'd like to fix this!

Reply:
Ah yes I guess it's because the mocks rejects to a `string` and `toThrowError` probably expects an actual `Error` instance?